features/writingsuggestions.yml

```
@@ -0,0 +1,7 @@
+name: writingsuggestions
+description: The `writingsuggestions` HTML attribute turns on or off a browser's writing suggestions, such as text completions or incremental autocompletions.
```

Continuing from https://github.com/web-platform-dx/web-features/pull/1579#discussion_r1724684775. I like the idea of making this concrete. What about something like this?

```suggestion
description: The `writingsuggestions` HTML attribute turns on or off a browser's writing suggestions. Writing suggestions vary by browser. For example, if turned on then a browser might show inline text completions accepted by pressing <kbd>Tab</kbd>.
```